prober/grpc.go

```diff
@@ -203,6 +200,7 @@ func ProbeGRPC(ctx context.Context, target string, module config.Module, registr
 	if serverPeer != nil {
 		tlsInfo, tlsOk := serverPeer.AuthInfo.(credentials.TLSInfo)
 		if tlsOk {
+			registry.MustRegister(probeSSLEarliestCertExpiryGauge, probeTLSVersion, probeSSLLastInformation)
```

Comment:
can we get a test for this, it will ensure we don't break this in future again.

Reply:
@electron0zero have you got any recommendations on how to achieve this? There doesn't seem to be an equivalent for the http probe.

I could create a new helper function that checks for the absence of registry results? Basically the inverse of this: 
https://github.com/prometheus/blackbox_exporter/blob/master/prober/utils_test.go#L35

Reply:
maybe checkMetrics can be used here? seehttps://github.com/prometheus/blackbox_exporter/blob/master/prober/utils_test.go#L173

if not, feel free to add a new helper to check for absence of a metric?

Reply:
@electron0zero I've added a helper function and a corresponding test, let me know if there's anything else required.